ivy/functional/ivy/sorting.py

```diff
@@ -138,12 +138,12 @@ def argsort(
 @handle_array_like
 def sort(
     x: Union[ivy.Array, ivy.NativeArray],
+    axis: int = -1,
```

Comment:
According to the array API standard, this is not correct. look here: https://data-apis.org/array-api/latest/API_specification/generated/signatures.sorting_functions.sort.html